librz/type/function.c

```diff
@@ -467,6 +467,9 @@ static bool callable_as_string(RzStrBuf *buf, const RzTypeDB *typedb, RZ_NONNULL
 			free(argstr);
 		}
 	}
+	if (callable->has_unspecified_parameters) {
```

Comment:
Please add a short unit test for this case in `test/unit/test_types.c` when you are done with the PR.